src/buffalonwb/metafile.yml

```diff
@@ -35,13 +35,6 @@ NWBFile:
   session_id: ADDME
   related_publications:
   session_start_time: 2017-05-04 11:23:13
-  slices:
   source_script: https://github.com/ben-dichter-consulting/buffalo-lab-data-to-nwb
   source_script_file_name: conversion_module.py
-  stimulus_notes:
```

Comment:
why were these removed? I think we should still have optional fields in the YAML file.

Reply:
any field can be included or removed! I removed those ones without info, but if the user add them, they'll be used automatically by the GUI/conversion

Reply:
Can we include all of the optional fields by default?

Reply:
If you want so, of course! It makes the yaml file less readable though, packed with not used info.